examples/github-tutorial/README.md

```diff
@@ -0,0 +1,14 @@
+# Juttle Tutorial: GitHub Dataset
+
+The *.juttle files here are Juttle programs that work with GitHub events dataset:
+
+- github_data_sample.json (small subset of the data)
+- github_data.json (full 30MB dataset, provided as zip archive)
+- github_metadata.csv (additional metadata file in CSV format)
+
+The programs assume that you have copied the data file to `/tmp/data.json` location.
```

Comment:
I think it would be good to explicitly list the command(s) that the user needs to do in order to get the data ready for the tutorial.